index.html

```diff
@@ -3,47 +3,434 @@
+        // Note that this is just the opposite of the above.
+      }
+    }
+
+    // Filter Occupations to Major Group Types
+    if (options.filtered) {
+        if (options.filtered === 'Mgt') {
+            data = data.filter(function (d) {return d.Group === 'Mgt'})
+        } else if (options.filtered === 'Sr') {
+            data = data.filter(function (d) {return d.Group === 'Sr'})
+        } else if (options.filtered === 'Drs') {
+            data = data.filter(function (d) {return d.Group === 'Drs'})
+        } else if (options.filtered === 'Admin') {
+            data = data.filter(function (d) {return d.Group === 'Admin'})
+        }
+    }
+
+    // Select only the schools from the current page (slice)
+    if (options.page) {
+        if (options.page === 0) {
+            data = data.slice(0, 21)
+        } else if (options.page === 1) {
+            data = data.slice(21,41)
+        } else if (options.page === 2) {
+            data = data.slice(41, 61)
+        } else if (options.page === 3) {
+            data = data.slice(61,81)
+        } else if (options.page === 4) {
+            data = data.slice(81,101)
+        } else if (options.page === 5) {
+            data = data.slice(101, 121)
+        } else if (options.page === 6) {
+            data = data.slice(121, 141)
+        }
+    }
+
+    var table = document.getElementById('data-table');
+    var tbody = table.getElementsByTagName('tbody')[0];
+    // Because getElementsByTagName returns an array of DOM nodes (technically, it's an HTMLCollection),
+    // we need to select the first item in the array with the [0].
+
+    // Remove any existing rows
+    tbody.innerHTML = '';
+
+    // Add a new row for each datum (item in the 'data' array)
+    data.forEach(function (d) {
+      var row = document.createElement('tr');
+      // This is like create a new <tr></tr> in HTML, but it doesn't yet exist in the DOM, so we can't
+      // see it. At the bottom of this forEach() loop, we append it to an element that is in the DOM,
+      // at which point it appears. We'll follow this same pattern for each of the <td></td> elements
+      // we create below.
+
+      var Occupation_name = document.createElement('td');
+      Occupation_name.innerHTML = d.Occupation_name;
+      row.appendChild(Occupation_name);
+
+      var Group_Name = document.createElement('td');
+      Group_Name.innerHTML = d.Group_Name;
+      row.appendChild(Group_Name);
+
+      var Men_Earn = document.createElement('td');
+      Men_Earn.innerHTML = d.Men_Earn;
+      row.appendChild(Men_Earn);
+
+      var Women_Earn = document.createElement('td');
+      Women_Earn.innerHTML = d.Women_Earn;
+      row.appendChild(Women_Earn);
+
+      var barWrapper = document.createElement('td');
+      var bar = document.createElement('div');
+      bar.setAttribute('class', 'bar');
+      var FemPercent = d.WE / d.ME;
+      bar.setAttribute('style', 'width: ' + (FemPercent * 100) + '%;');
```

Comment:
I like the idea of changing the color if the bar is over 100%. But I'm having trouble figuring out how to do this; does the code go here or in the style in the css?

I tried the following code right after I calculated FemPercent, but it just resulted in the bars not displaying and I don't know what to try next. Thanks for any further hints!

      if (FemPercent > 1) {
          bar.setAttribute('style', 'background: red');
      } else {
          bar.setAttribute('style', 'background: green');
      }